packages/integrations/cloudflare/src/index.ts

```diff
@@ -1,18 +1,31 @@
-import { createRedirectsFromAstroRoutes } from '@astrojs/underscore-redirects';
+import type { IncomingRequestCfProperties } from '@cloudflare/workers-types/experimental';
 import type { AstroAdapter, AstroConfig, AstroIntegration, RouteData } from 'astro';
+
+import { createRedirectsFromAstroRoutes } from '@astrojs/underscore-redirects';
+import { CacheStorage } from '@miniflare/cache';
+import { NoOpLog } from '@miniflare/shared';
+import { MemoryStorage } from '@miniflare/storage-memory';
+import { AstroError } from 'astro/errors';
 import esbuild from 'esbuild';
 import * as fs from 'node:fs';
 import * as os from 'node:os';
 import { sep } from 'node:path';
 import { fileURLToPath, pathToFileURL } from 'node:url';
 import glob from 'tiny-glob';
+import { getEnvVars } from './parser.js';
 
 export type { AdvancedRuntime } from './server.advanced';
 export type { DirectoryRuntime } from './server.directory';
 
 type Options = {
 	mode: 'directory' | 'advanced';
```

Comment:
```suggestion
	mode?: 'directory' | 'advanced';
```
When users want to specify `runtime`, they shouldn't need to specify `mode` as well.

Reply:
That makes sense, do we want to default to `advanced` mode?

Reply:
Yeah, that seems to be the implied default already.

Reply:
You're right, definitely something I'll change in a future PR, since it kinda should be more explicit.